filament/backend/include/backend/DriverEnums.h

```diff
@@ -84,40 +84,20 @@ enum class BufferUsage : uint8_t {
 };
 
 /**
- * Selects which buffers to clear at the beginning of the render pass, as well as which buffers
- * can be discarded and the beginning and end of the render pass.
+ * Defines a view port, that is the origin and extend of the clip-space.
```

Comment:
Nits:
- consistency with "viewport" vs. "view port".
- extent instead of extend
- "that is" => "which is"